09_Clean Code/praktikum/rewrite.js

```diff
@@ -0,0 +1,36 @@
+class Kendaraan {
+    constructor(){
+        this.kecepatanPerJam
+    }
+}
+
+class Mobil extends Kendaraan {
+    constructor(){
+        super()
+        this.kecepatanPerJam = 0
+    }
```

Comment:
Inheritance kecepatanPerJam punya class Kendaraan jadi useless, krn di class mobil this.kecepatanPerJam di declare dan di initial nilainya..